Jenkinsfile

```diff
@@ -47,7 +62,18 @@ pipeline {
         }
         stage('push') {
             steps {
-                sh "docker push ${AWS_ID_NUM}.dkr.ecr.${AWS_REG}.amazonaws.com/aline-account-microservice:latest"
+                sh "docker push ${AWS_ID_NUM}.dkr.ecr.${AWS_REG}.amazonaws.com/aline-account-microservice:pipeline-latest"
```

Comment:
Is the tag `pipeline-latest` correct? It doesn't follow the convention of your other microservice build scripts